README.md

```diff
@@ -8,3 +8,15 @@ This is not its documentation.
 - the **source code** is hosted at http://llvm.org/, and mirrored at
   https://github.com/llvm/llvm-project/tree/master/clang-tools-extra/clangd.
 - the **website source code** is at https://github.com/clangd/clangd.github.io/
+
+### Communication channels
+
+If you have any questions or feedback, you can reach community and developers
+through one of these channels:
+
+- quick chat #clangd room hosted on [LLVM's Discord
```

Comment:
I can't quite parse this fragment: just "#clangd room hosted..." or "Chat: #clangd room hosted..."?